src/main/java/org/broadinstitute/dsm/model/birch/RegisterBirchOrderCLI.java

```diff
@@ -0,0 +1,134 @@
+package org.broadinstitute.dsm.model.birch;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+import java.util.Scanner;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.apache.commons.dbcp2.PoolableConnection;
+import org.apache.commons.dbcp2.PoolingDataSource;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.dsm.careevolve.Authentication;
+import org.broadinstitute.dsm.careevolve.Covid19OrderRegistrar;
+import org.broadinstitute.dsm.careevolve.OrderResponse;
+import org.broadinstitute.dsm.careevolve.Patient;
+import org.broadinstitute.dsm.careevolve.Provider;
+import org.broadinstitute.dsm.cf.CFUtil;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.DdpKit;
+import org.broadinstitute.dsm.statics.ApplicationConfigConstants;
+import org.broadinstitute.dsm.statics.DBConstants;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+import org.elasticsearch.client.RestHighLevelClient;
+
+/**
+ * CLI utility for manually placing orders in CareEvolve (and birch)
+ * in response to BQMS tickets for kits that have been returned
+ * without UPS shipping info.
+ */
+public class RegisterBirchOrderCLI {
+
+    private static final String USAGE = "args in order are hruid, kitLabel, externalOrderNumber, and collectionTime\n" +
+            "for example java -Dconfig.file=... RegisterBirchOrderCLI P1234 TBOS-123 12345678910111213 06/30/2020 15:34\n" +
+            "Make sure you are on a broad network\n";
+    public static final String DATE_FORMAT = "MM/dd/yyyy HH:mm";
```

Comment:
@yufengwng your suggestion worked once I updated this from `hh` to `HH` (I've been putting in hours as 0-23).

Reply:
Noted! And it's helpful that the "usage" string shows an example of `15:34` hinting at 24-hr format!